common/navbar.js

```diff
@@ -6,10 +6,17 @@
         'chaise.authen',
         'ui.bootstrap'
     ])
-    .directive('navbar', ['$window', '$rootScope', 'Session', 'modalUtils', function($window, $rootScope, Session, modalUtils) {
+    .directive('navbar', ['$window', '$rootScope', 'Session', 'modalUtils', 'ConfigUtils', function($window, $rootScope, Session, modalUtils, ConfigUtils) {
+
+        var chaiseConfig = ConfigUtils.getConfigJSON();
 
     // One-time transformation of chaiseConfig.navbarMenu to set the appropriate newTab setting at each node
-        var root = chaiseConfig.navbarMenu = chaiseConfig.navbarMenu || {};
+        var root;
```

Comment:
There really wasn't a need to change this case. Setting the navBar menu to it's proper default means we can simplify this to:
```js
var root = $rootScope.chaiseConfig.navbarMenu
```